### PR TITLE
Feature/cisco ise pic

### PR DESCRIPTION
**Pull request by julsemaan**
# Description
Integration with Cisco ISE-PIC

# Impacts
Firewall SSO

# Delete branch after merge
YES

# Checklist
(REQUIRED) - [yes, no or n/a]
- [x] Document the feature
- [ ] Add unit tests
- [ ] Add acceptance tests (TestLink)

# NEWS file entries
## New Features
* Add single-sign-on support for Cisco ISE-PIC

### CODE REVIEW

**Comment by nqb** [comment]
Hi,

In AsciiDoc documents:
- use `=` for titles (see https://github.com/inverse-inc/packetfence/blob/devel/docs/PacketFence_Developers_Guide.asciidoc#titles)
- `ifdef::env-github[]` should not be necessary here

My 2 cents

**Comment by julsemaan** [comment]
@nqb, do we have a reference of the equivalents between the `=` style headers and the ones we used previously. Otherwise its hard for us to guess what levels to use.

File: html/pfappserver/lib/pfappserver/Form/Config/Firewall_SSO/CiscoIsePic.pm [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  );
+
+has_block definition =>
+  (
+   render_list => [ qw(id type port categories networks cache_updates cache_timeout username_format default_realm) ],
+  );
+
+=head1 COPYRIGHT
+
+Copyright (C) 2005-2019 Inverse inc.
```
Comment (fdurand) [inline html/pfappserver/lib/pfappserver/Form/Config/Firewall_SSO/CiscoIsePic.pm]: 2020

**Comment by nqb** [comment]
@julsemaan, not yet. You're right. I will try to provide a table.

**Comment by julsemaan** [comment]
@nqb, will you be able to provide the reference table so we can adjust this?

Otherwise we merge as-is and adjust this along with the other SSO documentations post-merge

**Comment by nqb** [comment]
Hello @julsemaan, see dc024843881939f662804e3cdf8fd7d19565403d

**Comment by julsemaan** [comment]
Adjusted based on comments

**Pull request merged**